MAUI/Cartesian-Charts/Zooming-and-panning.md

```diff
@@ -248,4 +248,132 @@ chart.ZoomPanBehavior = zooming;
 
 {% endhighlight %}
 
+{% endtabs %}
+
+## Selection zooming
+
+Selection zooming feature allows users to interactively choose a particular area of the chart and zoom in. By specifying [EnableSelectionZooming]() property to `true` as shown in the following code snippet, you can double tap and drag to select a range on the chart to be zoomed in. The default value of this property is false.
+
+{% tabs %}
+
+{% highlight xml %}
+
+<chart:SfCartesianChart>
+    <chart:SfCartesianChart.ZoomPanBehavior>
+        <chart:ChartZoomPanBehavior EnableSelectionZooming ="True"/>
+    </chart:SfCartesianChart.ZoomPanBehavior>
+    ...
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+ChartZoomPanBehavior zooming = new ChartZoomPanBehavior()
+{
+    EnableSelectionZooming = true
+};
+
+chart.ZoomPanBehavior = zooming;
+...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Selection zooming support in MAUI Chart](Zooming-and-panning_images/maui_selection_zooming.gif)
+
+### Selection rectangle customization
+
+You can customize the selection rectangle using the below properties.
+
+* [SelectionRectStrokeWidth]() - used to change the stroke width of selection rectangle
+
+* [SelectionRectStroke]() - used to change the stroke color of selection rectangle.
```

Comment:
![image](https://github.com/syncfusion-content/maui-docs/assets/92786382/844162a6-97fd-4630-b093-31ea297adf72)
To explain the property we used Gets or sets like for all other content so modify similar to this.

Reply:
modified property explanation